packages/eslint-plugin/docs/rules/no-useless-template-literals.md

```diff
@@ -0,0 +1,64 @@
+---
+description: 'Disallow unnecessary template literals.'
+---
+
+> 🛑 This file is source code, not the primary documentation location! 🛑
+>
+> See **https://typescript-eslint.io/rules/no-useless-template-literals** for documentation.
+
+This rule reports template literals that can be simplified to a normal string literal.
+
+## Examples
+
+<!--tabs-->
```

Comment:
[Docs] Many of our rule files intentionally have symmetry between the correct and incorrect examples in the docs. As in, if you switch between the two of them, you can see that lines switch between correct and incorrect code in-place. That's a pleasantry for readers so they can more quickly see the diffs between the two. Can you apply that here?

Reply:
that's a great idea, I like it!